language-design.md

```diff
@@ -62,7 +129,37 @@ the commenters?
 
 **Response**
 
+There are two interesting, and not necessarily conflicting, points of view in this article (and its comments).  On the one hand, there are the programmers, many of whom would assert that it is OK for there to be some kind of overhead: 
 
+> Our languages may be esoteric and the tools hard to master, but show me any other technical profession where tools are straightforward to learn and mastery comes easily. [Comment by "Strengthiness" ([Source](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-93e75ca0-9b5b-11e4-a3e4-4930d33952b2))] [Pavlus, 2012].  
+
+There are even those who think that it is good for there to be this overhead:
+
+> The issue with writing software is not being able to program; it's being able to program \*well\*.
+
+> There are a number of languages out there which have been specifically designed to be easier to write for novice programmers.  
+> ...  
+>  
+> I guess most companies have got somewhere ... [a] program that they rely on for some task but which is woefully inadequate, or worse, a security minefield, because it was written by someone who didn't really understand the technical issues. [Comment by "Simon" ([Source](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-95ed5400-5bc1-11e1-921b-9794fc737ae8)) [Pavlus, 2012].  
+
+ 
+They raise a good point - programmers handle a lot of really important, really sensitive things that should certainly be done as well as possible.  Furthermore, the purpose of the tool is to design it to be usable by the programmers once they have learned their craft - something designed for novices may end up being used exclusively by novices. 
```

Comment:
While I agree that being able to program _well_ is important for high-traffic programs, I don't think that all programs _need_ to be well-written. A language designed for and exclusively used by novices isn't inherently bad if people have found a language in which they feel comfortable expressing themselves and empowers them.